pkg/daemon/daemon.go

```diff
@@ -232,13 +236,13 @@ func (d *Daemon) SharedMount(rafs *Rafs) error {
 		}
 		return nil
 	case config.FsDriverFusedev:
-		return d.sharedFusedevMount(rafs)
+		return d.sharedFusedevMount(rafs, authCache)
 	default:
 		return errors.Errorf("unsupported fs driver %s", d.States.FsDriver)
 	}
 }
 
-func (d *Daemon) sharedFusedevMount(rafs *Rafs) error {
+func (d *Daemon) sharedFusedevMount(rafs *Rafs, authCache *auth.Cache) error {
```

Comment:
Erofs/fscache fs driver should also need the auth

Reply:
Fixed.